umbraco/Umbraco.CMS.Backoffice
# Feature: Add User Group Alias

> [!NOTE]  
>Merges together with: https://github.com/umbraco/Umbraco-CMS/pull/16154



## Description

This feature adds the user group alias to the UI.

What to test:
* make sure you can change the alias when allowed
* make sure you can't change the alias when not allowed (ex: admin, sensitive-data)
* make sure you can create a new group with an alias + change it again.



## Types of changes




## Motivation and context



## How to test?

## Screenshots (if appropriate)

## Checklist
